doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Term definitions
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting.
+
+An **inventory plan** is a set of rules that define the content that belongs to a theme. 
```

Comment:
But in the users' domain model, Content Items _do_ move across Themes - shouldn't we be ready to model that in code, too?